### PR TITLE
mist: add page

### PR DESCRIPTION
**Pull request by macownersclub**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 2.1**

### CODE REVIEW

File: pages/osx/mist.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# mist
+
+> MIST - macOS Installer Super Tool.
+> Automatically download macOS Firmwares / Installers.
```
Comment (kbdharun) [inline pages/osx/mist.md:4]:
```suggestion
> Automatically download macOS Firmwares/Installers.
```
We generally prefer this across our pages.

File: pages/osx/mist.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- List all available macOS Installers for Intel Macs, including betas, also including Universal Installers for macOS Big Sur and later:
+
+`mist list installer --include-betas`
+
+- List only the latest macOS Sonoma Installer for Intel Macs, including Universal Installers for macOS Big Sur and later:
+
+`mist list installer --latest "macOS Sonoma"`
+
+- List + Export macOS Installers to a CSV file:
```
Comment (kbdharun) [inline pages/osx/mist.md:27]:
```suggestion
- List and export macOS Installers to a CSV file:
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Review by spageektti — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**